src/stores/tenant.rs

```diff
@@ -373,6 +378,18 @@ impl TenantStore for PgPool {
 
         Ok(res)
     }
+
+    async fn suspend_tenant(&self, id: &str, reason: &str) -> Result<()> {
+        sqlx::query_as::<sqlx::postgres::Postgres, Tenant>(
+            "UPDATE public.tenants SET suspended = true, suspended_reason = $2::text WHERE id = \
+             $1 RETURNING *;",
```

Comment:
I think `RETURNING *` is not needed?

```suggestion
             $1;",
```

Reply:
good catch, removed